source/includes/_instruments.md

```diff
@@ -327,3 +327,165 @@ Http Status:
 | 400 Bad Request  | The request was malformed.  See response body    | None       | n/a              |
 | 401 Unauthorized | No auth provided, auth failed, or not authorized | None       | n/a              |
 | 404 Not Found    | No allocation with this UUID exists              | None       | n/a              |
+
+
+
+## Instrument Reporting
+
+The [`POST /instruments/reporting`](/#settlement-equity-post-instruments-reporting) endpoint allows platforms to report their system's view on equity ownership for reconcillation purposes.
+
+Equity ownership can be communicated to Goji as a snapshot via multiple calls to [`POST /instruments/reporting`](/#settlement-equity-post-instruments-reporting) describing investor holdings on a per-instrument basis. 
+
+⚠️ All instruments that are part of the same snapshot should share the same `reportId`.
+
+
+## Instrument Reporting Model
+
+### Required Attributes
+
+All of the following attributes of the Instrument Reporting Model are considered 'required' attributes. 
+
+⚠️ In the case of an instrument that has not yet been issued or traded to investors - the holdings section can be left empty.
```

Comment:
Will it even need to be reported at all? Either way fine but could be useful clarifying.

Reply:
i think it's probably clearer if they just report all instruments regardless.